.github/workflows/rahim-experiments.yml

```diff
@@ -0,0 +1,133 @@
+name: Random Github Action Experiments
```

Comment:
This will not make into final.  Just to be used to run the steps faster than using e2e-detox-pr workflow.